### DISCUSSION TITLE
[Q&A] build之后 启动 白屏 Application error: a client-side exception has occurred (see the browser console for more information).[Bug]

### DISCUSSION THREAD

**Discussion by dashixiong-11** [lobehub/lobehub#975] [Development Question] [answered]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

build之后 启动 白屏 Application error: a client-side exception has occurred (see the browser console for more information).
dev 没有这个问题 
![image](https://github.com/lobehub/lobe-chat/assets/39214393/de54280e-6c2a-4ae9-88da-e7547a0ed5d7)
![image](https://github.com/lobehub/lobe-chat/assets/39214393/c68dbb2b-6fbc-4d48-9c84-d8246470737e)
![image](https://github.com/lobehub/lobe-chat/assets/39214393/deacc811-113d-4cbe-a8b3-458f706ff577)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
本地 antd 版本问题吧。重新装下依赖应该就好了。refs: 

- https://github.com/swc-project/swc/issues/8475

[a comment by an automated account is not shown]
